index.html

```diff
@@ -1,115 +1,276 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>Baystation 12</title>
+		<link rel="icon" href="/favicon.ico">
+		<style>
```

Comment:
That's the plan! _After_ it's good to go. Up/down is faster than tabs for scratch dev IMO.